elixir/apps/web/lib/web/live/gateways_live/show.ex

```diff
@@ -147,7 +178,7 @@ defmodule Web.GatewaysLive.Show do
               Deployment method
             </th>
             <td class="px-6 py-4">
-              Docker
+              TODO: Docker
```

Comment:
We don't have this data, so we can remove it for now. @jamilbk what is the use case to show it?

Reply:
Two reasons to keep this:

1. As an administrator, I would like to know which deployment method I used so that I can quickly know which upgrade method to apply without having to poke around my servers.
2. We want telemetry metrics around this

Reply:
We sent this telemetry in the portal today using this method: https://stackoverflow.com/questions/20010199/how-to-determine-if-a-process-runs-inside-lxc-docker

Reply:
Sounds good 👍

Reply:
@conectado please include this as part of User-Agent :).

Reply:
@AndrewDryga Ah, good call!